files/src/main/java/com/fincity/saas/files/service/AbstractFilesResourceService.java

```diff
@@ -739,6 +745,253 @@ public Mono<FileDetail> create(String clientCode, String uri, FilePart fp, Strin
 				.contextWrite(Context.of(LogUtil.METHOD_NAME, "AbstractFilesResourceService.create"));
 	}
 
+	public Mono<FileDetail> imageUpload(String clientCode, String uri, FilePart fp, String fileName, Boolean override,
```

Comment:
It is very confusing with multiple variables with similar names. Please refactor and make it more readable.